test/go.test.ts

```diff
@@ -298,14 +304,39 @@ It returns the number of bytes written and any write error encountered.
 		}).then(() => done(), done);
 	});
 
+	test('Test Generate unit tests squeleton for a function', (done) => {
+		getGoVersion().then(version => {
+			if (version.major === 1 && version.minor < 6) {
+				// gotests is not supported in Go 1.5, so skip the test
+				return Promise.resolve();
+			}
+
+			let uri = vscode.Uri.file(path.join(generateFunctionTestSourcePath, 'generatetests.go'));
+			return vscode.workspace.openTextDocument(uri).then(document => {
+				return vscode.window.showTextDocument(document).then((editor: vscode.TextEditor) => {
+					assert(vscode.window.activeTextEditor, 'No active editor');
+					let selection = new vscode.Selection(5, 0, 6, 0);
+					editor.selection = selection;
+					return generateTestCurrentFunction().then((result: boolean) => {
+						assert.equal(result, true);
```

Comment:
Ideally, I'd like to check whether the test file was created with the test function. You don't have do it now, but if you have some time, I'd appreciate it.